docs/source/distributed.rst

```diff
@@ -428,6 +428,22 @@ Collective functions
 
     :class:`~torch.distributed.ReduceOp` is recommended to use instead.
 
+Profiling Collective Communication
+-----------------------------------------
+
+Note that you can use ``torch.profiler`` or ``torch.autograd.profiler`` to profile collective communication and point-to-point communication APIs mentioned here. All out-of-the-box backends (``gloo``,
```

Comment:
IIRC, the latter `torch.autograd.profiler` probably will be deprecated in favor of `torch.profiler`, so we can mention this:

``torch.profiler`` (recommended, only available after 1.8.1)

Reply:
Yes good point, will add a note